src/sentry/snuba/metrics/query.py

```diff
@@ -378,8 +376,8 @@ def validate_granularity(self) -> None:
 
     def validate_interval(self) -> None:
         if self.interval is not None:
-            if self.use_case_key == UseCaseKey.RELEASE_HEALTH or (
-                self.use_case_key == UseCaseKey.PERFORMANCE and not self.include_series
+            if self.use_case_id is UseCaseID.SESSIONS or (
+                self.use_case_id is UseCaseID.TRANSACTIONS and not self.include_series
```

Comment:
```suggestion
                self.use_case_id in {UseCaseID.TRANSACTIONS, UseCaseID.SPANS} and not self.include_series
```

Reply:
It was suggested by @iambriccardo that we should [hold off on any changes regarding spans for now](https://github.com/getsentry/sentry/pull/51776#issuecomment-1617538339). And the telemetry team will add it later.